### DISCUSSION TITLE
Add ability for Web Clipper to Bookmark a page.

### DISCUSSION THREAD

**Discussion by drtobbyas** [standardnotes/forum#3327] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Not, really

**Describe the solution you'd like**

I would to see an additional feature on the Web Clipper Extension to be able to bookmark a page or save a web page as a bookmark with only the Title, Link and head Description of the page populated on the note app instead of saving the whole content of the page as an article. Sometimes I want to save the page as a Bookmark just like a Chrome bookmark would save the link and title of the page. 

It would be a great feature to be able to organize my bookmarks alongside my docs and notes.

This is a feature that can be added to the Web Clipper Extension. I don't think much development is required on the note app because saving a page as a bookmark will only add the Head Title of the page as a Note Title, Link and description of the page will be populated in the content of the note.

I can raise a PR to this effect if the proposal is approved

**Comment by effieeee** [comment]
Hi @drtobbyas, you can still raise a PR for this for the team to review! The clipper is in the repo link below:
https://github.com/standardnotes/app/tree/main/packages/clipper
